pylint_odoo/checkers/no_modules.py

```diff
@@ -387,8 +394,33 @@ def visit_call(self, node):
                 isinstance(node.parent.parent, astroid.ClassDef):
             has_help = False
             args = misc.join_node_args_kwargs(node)
+            index = 0
             for argument in args:
                 argument_aux = argument
+                # Check this 'name = fields.Char("name")'
+                if (isinstance(argument, astroid.Const) and
+                    ((index == 4 and
+                      argument.parent.func.attrname == 'Many2many') or
```

Comment:
What about use a GLOBAL dict variable?
I mean,

```python
STRING_ARG_INDEX = {
    'Many2many': 4,
    'One2many': 2,
    'Many2one': 1,
    'Reference': 1
    'Selection': 1
}
```

and implementing like as:
```python
if STRING_ARG_INDEX.get(argument.parent.func.attrname, 0) == index
```